pandas/tests/generic/test_generic.py

```diff
@@ -948,3 +948,10 @@ def test_deprecated_get_dtype_counts(self):
         df = DataFrame([1])
         with tm.assert_produces_warning(FutureWarning):
             df.get_dtype_counts()
+
+    @pytest.mark.parametrize("axis", [0, 1, "index", "columns", "rows"])
```

Comment:
move to pandas/tests/frame/test_missing.py near the other interpolate tests.

Reply:
Moved the test to pandas/tests/frame/test_missing.py::TestDataFrameMissingData::test_interp_rowise.